.github/workflows/collect-deps.yml

```diff
@@ -0,0 +1,43 @@
+name: Collect ubuntu browser dependencies
```

Comment:
Scheduled action:
- Collects ubuntu dependencies for new browser versions
- Creates a PR to add dependency if there is any new library
- If there are no extra dependencies, do nothing.
- If there is extra browser, but no new dependencies, creates PR to update cache, so it would not need to download the browser later